gcp/modules/gpii-preferences/resources/preferences-network-policy.yaml

```diff
@@ -0,0 +1,32 @@
+piVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: preferences-network-policy
+  namespace: gpii
+spec:
+  podSelector:
+    matchLabels:
+      app: preferences
+  policyTypes:
+  - Ingress
+  - Egress
+  ingress:
```

Comment:
We've talked about ceasing to expose preferences server to the public internet, but we don't have a plan in place yet. Meanwhile, I think this is going to break Locust / smoke tests. If it does, how should we proceed?

Reply:
I need to double check this after another clean redeployment - but I believe inbound traffic through the ingress is still preserved with these rules.

Reply:
Looking at this again, I guess it's not surprising to connect an Ingress to something and expect the something to allow a connection from that Ingress :).

Reply:
You'll need to add policy allowing the traffic from the nginx ingress controller pods, otherwise it'll be blocked.